sshd/zenith/sshd/tunnel.py

```diff
@@ -169,33 +179,31 @@ def get_tunnel_config(timeout_secs):
                 else:
                     input_lines.append(line)
     except TimeoutError:
-        print(
-            "[SERVER] [ERROR] Timed out negotiating tunnel configuration",
-            file = sys.stderr
-        )
-        sys.exit(1)
+        logger.critical("Timed out negotiating tunnel configuration")
```

Comment:
I think this should be error?

Reply:
hang on, don't we need to still re-raise the exception here?

Reply:
There is a handler registered that raises an exception and exits with an error code when a critical log is sent.